jkube-kit/build/service/jib/pom.xml

```diff
@@ -27,13 +27,22 @@
   <artifactId>jkube-kit-build-service-jib</artifactId>
 
   <name>JKube Kit :: Build :: Service :: JIB</name>
+  <properties>
+    <version.guava>33.4.0-jre</version.guava>
+  </properties>
```

Comment:
To make maintenance easier, the dependencyManagement section and version property should be kept in the root/parent pom.
We should then add the dependency here without the version